astropy/units/tests/test_units.py

```diff
@@ -688,3 +688,13 @@ def test_fractional_rounding_errors_simple():
     assert isinstance(x.powers[0], Fraction)
     assert x.powers[0].numerator == 6
     assert x.powers[0].denominator == 5
+
+
+def test_enable_unit_groupings():
+    from ...units import cds
+    with cds.enable():
+        pass
```

Comment:
Shouldn't this also test that the enabling actually did anything? In this case that a cds-specific unit is present and a general-specific one is not? (same for imperial, except there of course both should be present).